eng/pipelines/common/templates/wasm-library-aot-tests.yml

```diff
@@ -1,7 +1,7 @@
 parameters:
   alwaysRun: false
   extraBuildArgs: ''
-  extraHelixArgs: ''
+  extraHelixArguments: ''
```

Comment:
Why the rename? Are you planning to expand `extraBuildArgs` to `extraBuildArguments` as well?

Reply:
unification, so that copy&paste works without thinking about the context. All places now have `extraHelixArguments` an `extraBuildArgs`

Reply:
I just don't understand why have `args` for one and `arguments` for other 🤷‍♂️

Reply:
you are asking me to fix something which is far beyond scope of this PR